src/css/jwplayer/imports/title.less

```diff
@@ -10,34 +9,33 @@
 // if we set color on the parent, it is overwritten by the reset
 .jw-title-primary,
 .jw-title-secondary {
+    color: @white;
```

Comment:
would this prevent titles from being customizable ?

Reply:
Nope, this is just sorting the existing props and swapping the `#fff` with the Less variable `@white`, which is already `#fff`